ruby/test/test_helper.rb

```diff
@@ -0,0 +1,30 @@
+require 'simplecov'
+require 'simplecov-cobertura'
+
+
+SimpleCov.start do
+  formatter SimpleCov::Formatter::MultiFormatter.new([
+    SimpleCov::Formatter::HTMLFormatter, # Add HTML report for viewing/review.
+    SimpleCov::Formatter::CoberturaFormatter # For CI
+    ])
+end
+
+require 'minitest/autorun'
+require 'minitest/reporters'
+
+if ENV['CI_COMMIT_TAG']
+  run_id = ENV['CI_COMMIT_TAG']
+elsif ENV["CI"]
+  run_id = "#{ENV['CI_PIPELINE_ID']}-#{ENV['CI_COMMIT_SHORT_SHA']}"
```

Comment:
Please check these variables are valid for Github, these look like Gitlab variable names